sale_order_invoicing_finished_task/views/product_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record
+        id="product_template_form_view_invoice_policy_inherit_sale_project"
+        model="ir.ui.view"
+    >
+        <field name="model">product.template</field>
+        <field
+            name="inherit_id"
+            ref="sale_project.product_template_form_view_invoice_policy_inherit_sale_project"
+        />
+        <field name="arch" type="xml">
+            <field name="project_id" position="before">
+                <field
+                    name="invoicing_finished_task"
+                    invisible="type != 'service' or service_tracking not in ['task_global_project', 'task_in_project']"
+                />
+            </field>
```

Comment:
Improve UI
![image](https://github.com/user-attachments/assets/6cb3db1b-b767-4c25-b83d-efd72804ef7a)
With option `Create Order = Task` selected
![image](https://github.com/user-attachments/assets/6b8b4223-3a7e-4242-893a-9637b0180c42)

Reply:
I have made the changes, and it is now displaying correctly.

Reply:
Hi, sorry but i not see the field, can you please check it
![image](https://github.com/user-attachments/assets/5980d343-442e-4fba-afac-828aea1b7334)